.env

```diff
@@ -1,3 +1,6 @@
 export FLASK_APP="run.py"
 export FLASK_ENV="production"
-export FLASK_DEBUG="True"
\ No newline at end of file
+export FLASK_DEBUG="True"
+export DEV_DATABASE_URL="dbname='politico-api' user='postgres' host='localhost' password='postgres'"
+export TEST_DATABASE_URL="dbname='test-politico-api' user='postgres' host='localhost' password='postgres'"
+export PROC_DATABASE_URL="dbname='d6tlj0h7cqmb9k' user='hvpwhvpxbuwgbj' host='ec2-54-235-159-101.compute-1.amazonaws.com' password='913f3c75858bde9b9ee8c2d77bdf0fd0f0c96c4b52d56e7e9b6b2da9c3cc4eda'"
```

Comment:
@lenileiro The `.env` should not be committed to git as it exposes sensitive information. Kindly check on this link to find out how to deal with this https://github.com/bkeepers/dotenv/issues/67.

Reply:
I will add it to the .gitignore file then make a commit with the changes